dazel/lib/src/bazelify/build.dart

```diff
@@ -154,6 +161,15 @@ class BuildFile {
       buffer.writeln('load("$_vmBzl", "dart_vm_binary")');
       buffer.writeln();
     }
+    final buildersUsed =
+        libraries.expand((l) => l.builders?.keys ?? const <String>[]);
+    if (buildersUsed.isNotEmpty) {
+      buffer
+        ..writeln('# Dazel: Using codegen.')
+        ..writeln(
+            'load(":codegen.bzl", ${buildersUsed.map((b) => '"$b"').join(', ')})')
```

Comment:
Switched up to per-package skylark files and fixed the load to start at the root.